wm_extra/views.py

```diff
@@ -419,9 +419,11 @@ def geoexplorer2worldmap(config, map_obj, layers=None):
             if layer_config['local'] == True:
                 # WM local layer to process
                 if 'styles' not in layer_config:
-                    layer = Layer.objects.get(typename=layer_config['name'])
```

Comment:
This code was needed because some WM layers are missing the styles information and the layer fails to display. Why is it needed to remove it?